### DISCUSSION TITLE
Filtering using back link.

### DISCUSSION THREAD

**Discussion by atulhajare** [Budibase/budibase#9147] [Help]
Hi,

I have used breadcrumb in my screen and while clicking on that breadcrumb I should navigate to the clicked screens for e.g. let's say I have a breadcrumb like below:

Data1 > Data 2 > Data 3

So when I click on let's say Data 2 it should navigate me to the screen  of Data 2 and it should filter the data of the screen based on Data 2 (it is a string taken from the last screen)  

so need help on this how can I achieve this functionality?

**Comment by melohagan** [comment]
Hey @atulhajare 

Have you tried using Form steps? https://docs.budibase.com/docs/form-steps

You may find this custom Stepper component useful in this case: https://github.com/poirazis/bb-plugin-Stepper
Although it doesn't appear to have a click handler though.
